ui/components/component-library/text/text.scss

```
@@ -55,6 +55,11 @@ $text-variants: (
       @else {
         font-weight: $weight;
       }
+
+      // Rule for strong tag inside Text component
+      strong {
+        font-weight: bold;
```

Use the design token here
```suggestion
        font-weight: var(--font-weight-bold);
```